doc/qd-channel.rst

```diff
@@ -10,40 +10,52 @@ Example Module Documentation
+
+The current approach has the following limitations:
+
+* The folder structure is rigid, an incorrect structure should break the simulation before it even starts, but possible silent fails are not ecluded
+* Time steps are considered to be of equal duration and computed from the scenario itself
+* The channel is considered constant over a timestep, with no support to fast fading for the moment
+* Polarization is not supported
+* Mobility is not fully supported on ns-3. While the ray-tracing software can support mobility, only its effects are imported into ns-3 in the form of a varying channel. Currently, only ``ConstantPositionMobilityModel`` is supported. Please check the `Usage`_ section on why and how this works.
 
 References
 ==========
 
-Add academic citations here, such as if you published a paper on this
-model, or if readers should read a particular specification or other work.
+Some works using this module already exist:
+
+* M. Lecci, P. Testolina, M. Polese, M. Giordani, M. Zorzi, "Accuracy vs. Complexity for mmWave Ray-Tracing: A Full Stack Perspective," preprint available: `arXiv:2007.07125 <https://arxiv.org/abs/2007.07125>`_
+* M. Lecci, T. Zugno, S. Zampato, M. Zorzi, "A Full-Stack Open-Source Framework for Antenna and Beamforming Evaluation in mmWave 5G NR," preprint available: `arXiv:2011.05800 <https://arxiv.org/abs/2011.05800>`_
+
+Furthermore, theoretical papers regarding the ray-tracing simulator, the Quasi-Deterministic (QD) channel model, and measurements to validate the channel can be found here:
+
+* M. Lecci, M. Polese, C. Lai, J. Wang, C. Gentile, N. Golmie, M. Zorzi, "Quasi-Deterministic Channel Model for mmWaves: Mathematical Formalization and Validation," to be presented at IEEE IEEE Global Communications Conference (GLOBECOM), Dec. 2020, Taipei, Taiwan, available here: `arXiv:2006.01235 <https://arxiv.org/abs/2006.01235>`_
+* M. Lecci, P. Testolina, M. Giordani, M. Polese, T. Ropitault, C. Gentile, N. Varshney, A. Bodi, and M. Zorzi, "Simplified ray tracing for the millimeter wave channel: A performance evaluation", in Proceedings of the Workshop on Information Theory and Applications (ITA), Feb. 2020, San Diego, USA, available here: `arXiv:2002.09179 <https://arxiv.org/abs/2002.09179>`_, DOI: 10.1109/ITA50056.2020.9244950
+* C. Gentile, P. B. Papazian, R. Sun, J. Senic, and J. Wang, “Quasi- Deterministic Channel Model Parameters for a Data Center at 60 GHz,” IEEE Antennas and Wireless Propagation Letters, vol. 17, no. 5, pp. 808–812, May 2018, DOI: 10.1109/LAWP.2018.2817066
+* C. Lai, R. Sun, C. Gentile, P. B. Papazian, J. Wang, and J. Senic, “Methodology for multipath-component tracking in millimeter-wave channel modeling,” IEEE Transactions on Antennas and Propagation, vol. 67, no. 3, pp. 1826–1836, Mar. 2019, DOI: 10.1109/TAP.2018.2888686
+
+
+Please cite our work if you find it interesting!
```

Comment:
All of them? :)

Reply:
what do you suggest? the TWC paper is still under review..

Reply:
I would suggest to suggest the TWC, maybe we can also think about a paper that explicitly describes this implementation.